backend/database/databaseHelpers.js

```diff
@@ -0,0 +1,13 @@
+const DBimports = require('./database');
+
+const createConsumer = (user) => {
```

Comment:
do not merge any code that you arent sure works...

the code in here will not work properly...rememeber....if you write code...you need to obe able too test your code...everything on this page has erroors